Open confirm window before recoding data values

tokland opened the pull request:
Closes #38 

![screenshot from 2018-05-22 10-43-05](https://user-images.githubusercontent.com/24643/40352243-a193c9e2-5dae-11e8-93ae-f13a8e572794.png)

tokland committed:
Open confirm window before recoding data values

adrianq committed:
change strings and cosmetics

adrianq committed:
more styling

adrianq committed:
string and cosmetics

adrianq committed:
remove colon

adrianq reviewed (commented):
@tokland I made some comestics changes. Seems to be working fine but I have detected sth strange: first click on "Update" button do not trigger any action or confirmation dialog. You need to click twice in order to get the confirmation dialog. This happens just the very first time you click on the button. Can you please have a look at it?

tokland committed:
Move ng-if confirmation to inner div (fixes first click on update)

tokland committed:
Use literal UPDATE_CONFIRMATION_TITLE in confirmation window

tokland commented:
You're right, it happened on the first click. I thought it was me being clumsy with the mouse. Fixed. 

I see you created a literal UPDATE_CONFIRMATION_TITLE, but was not used, I updated it.

adrianq commented:
Works like a charm now!
